backend/routes/forms.js

```diff
@@ -102,10 +110,14 @@ router.delete("/:formId/sections/:sectionId", (req, res) => {
 // NOTE: The entire sections array is overwritten, req.body should include
 //       the _id for each section if _id is required to stay constant
 router.patch("/:formId/sections", (req, res) => {
-  db[req.headers.database].forms.findByIdAndUpdate(
-    req.params.formId,
+  db[req.headers.database].forms.findOneAndUpdate(
+    { formId: req.params.formId },
```

Comment:
Note: 
formID will be changed to programId in a future PR. (that is what it is). 

Other endpoints use the _id of the form so we are doing it to avoid confusion. 

Terminology will be updated elsewhere as well.